### PR TITLE
tag hierarchy

### PR DESCRIPTION
**Pull request by ReenaOmana**
created tag hierarchy

### CODE REVIEW

File: ui.content/src/main/content/META-INF/vault/filter.xml [hunk 1]
```diff
@@ -5,4 +5,5 @@
     <filter root="/content/dam/hclecomm" mode="merge"/>
     <filter root="/content/experience-fragments/hclecomm" mode="merge"/>
 	<filter root="/oak:index/hclecommLuceneIndex" mode="merge"/>
+    <filter root="content/cq:tags/ecommtags" mode="merge"/>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/META-INF/vault/filter.xml]: @ReenaOmana: please add / before content else filter wont work

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description=""
+    jcr:primaryType="cq:Tag"
+    jcr:title="ecommTags"
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml]: @ReenaOmana : jcr:title="hcl-eCommm"

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    sling:resourceType="cq/tagging/components/tag">
+    <gender/>
+    <clothing/>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml]: @ReenaOmana : Pl add jcr:description="HCL eComm Tags"

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/apparals/footwear/.content.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:description=""
+    jcr:primaryType="cq:Tag"
+    jcr:title="footwear"
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/apparals/footwear/.content.xml]: @ReenaOmana  : jcr:title="Footwear" , jcr:title for all tags to have First letter caps

File: ui.content/src/main/content/META-INF/vault/filter.xml (hunk 1 above)
Comment (ReenaOmana) [reply ui.content/src/main/content/META-INF/vault/filter.xml]: Have made the changes

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml (hunk 2 above)
Comment (ReenaOmana) [reply ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml]: This has been added

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml (hunk 3 above)
Comment (ReenaOmana) [reply ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/.content.xml]: Description has been added

File: ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/apparals/footwear/.content.xml (hunk 4 above)
Comment (ReenaOmana) [reply ui.content/src/main/content/jcr_root/content/_cq_tags/ecommtags/apparals/footwear/.content.xml]: These changes are taken care

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Pull request merged**